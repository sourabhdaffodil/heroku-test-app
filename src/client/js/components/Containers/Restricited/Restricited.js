import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const Div = styled.div`
  max-width: 1012px;
  width: 100%;
  box-sizing: border-box;
  padding: 0 12px;
  margin: 0 auto;
`;

const Container = props => (
  <Div {...props}>
    { props.children }
  </Div>
);

Container.propTypes = {
  children: PropTypes.any.isRequired,
};

export default Container;
