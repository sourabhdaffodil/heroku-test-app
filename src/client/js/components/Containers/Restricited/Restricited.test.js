import React from 'react';
import { shallow } from 'enzyme';
import Restricited from './Restricited';

describe('Restricited container component', () => {
  it('Must render a non empty container', () => {
    const wrapper = shallow(
      <Restricited>
        <p>Some content</p>
      </Restricited>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
