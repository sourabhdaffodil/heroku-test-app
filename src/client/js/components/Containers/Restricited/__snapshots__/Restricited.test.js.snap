// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restricited container component Must render a non empty container 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Container>
    <p>
      Some content
    </p>
  </Container>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": <p>
        Some content
      </p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Some content",
      },
      "ref": null,
      "rendered": "Some content",
      "type": "p",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <p>
          Some content
        </p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Some content",
        },
        "ref": null,
        "rendered": "Some content",
        "type": "p",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
